core/base/requirements-dev.txt

```diff
@@ -16,6 +16,7 @@ Flask-Login
 flask-marshmallow
 Flask-Migrate
 Flask-SQLAlchemy<3
+Flask-script
```

Comment:
Is this used anywhere? If so, why doesn't it appear in requirements-prod.txt?

Reply:
I don't think so, removing